### DISCUSSION TITLE
How to change font for node Details to Markdown type?

### DISCUSSION THREAD

**Discussion by reregaga** [freeplane/freeplane#805] [Help] [answered]
I have node and its Content Type for node Details is Markdown, and I don't know where to change font for preview. For example, on left node the Content Type is HTML and there in editor you can change font for preview, there is no such thing for Markdown in the text editor from `Edit node details in dialog`:
>![compare](https://user-images.githubusercontent.com/76629445/200111944-4aad21c7-68bb-4346-a11e-25491eb067cd.png)

I found where to change font for editor, but I couldn't find one for the preview:
>![md setting](https://user-images.githubusercontent.com/76629445/200111925-8c151e0f-3ecd-4796-8e32-1701cb56ebe4.png)

**Comment by macmarrum** [comment, marked as answer]
Details is actually a Style; its properties, including font family, can be changed in `Format->Manage Styles->Edit styles` -> `Predefined Styles` -> `Details`

Please see also https://docs.freeplane.org/user-documentation/css-for-html-or-markdown.html

**Reply by reregaga** [reply]
Thank you, that link:
https://docs.freeplane.org/user-documentation/css-for-html-or-markdown.html#css-for-details-and-note
helped. 

Unfortunately `*` CSS selector is not recognized, so I had to use `body`, probably should replace example from preview with example from link:
https://docs.freeplane.org/user-documentation/css-for-html-or-markdown.html#example-of-markdown-with-css
There are more elements presented including `code` element (does not exist in preview in version `1.10.4`):
>![preview](https://user-images.githubusercontent.com/76629445/200114961-7f0403fb-95c8-4151-b611-892d898d8ed3.png)
